spec/models/primary_retake_english_maths.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
```

Comment:
Missing `_spec` from the suffix of this file - noticed this in a few other places may be worth doing a quick audit?

Reply:
Yes, am going cross eyed from the number of files...